src/Data/provideLocalStorageDataClass.ts

```diff
@@ -142,6 +145,34 @@ export function provideLocalStorageDataClass(
   }
 }
 
+export function searchLocalStorageDataClasses(
+  search: string,
+  blackListEntities: string[] = [],
+): Array<{ _id: string; entity: string; title: string }> {
+  const results: Array<{ _id: string; entity: string; title: string }> = []
+
+  const lowerCaseSearchTerm = search.toLowerCase()
+  const matchesSearchTerm = (value: unknown) =>
+    typeof value === 'string' &&
+    value.toLowerCase().includes(lowerCaseSearchTerm)
+
+  recordKeys.forEach(({ className: entity, recordKey }) => {
```

Comment:
* [ ] FYI we're mixing and renaming `className` and `entity`/`entities` for the same thing. Can we stick to either one or the other for better readability?